docs/cookbook/prompt_engineering/chaining_based/prompt_paraphrasing.md

```diff
@@ -0,0 +1,97 @@
+# Prompt Paraphrasing: Generating Diverse Prompts for LLM Testing and Evaluation
+
+[Prompt Paraphrasing](https://direct.mit.edu/tacl/article/doi/10.1162/tacl_a_00324/96460/How-Can-We-Know-What-Language-Models-Know) is not a prompt engineering technique, but rather a prompt generation technique used to create ensembles of prompts for testing or other prompt engineering techniques. In this example, we cover a specific method of generating prompts mentioned in the paper whereby a prompt is translated into $B$ versions in another language, then backtranslated into $B^2$ versions to English.
+
+??? tip "Mirascope Concepts Used"
+
+    - [Prompts](../../../learn/prompts.md)
+    - [Calls](../../../learn/calls.md)
+    - [Response Models](../../../learn/response_models.md)
+
+## Implementation
+
+Let's implement the Prompt Paraphrasing technique using Mirascope:
+
+```python
+from pydantic import BaseModel, Field
+
+from mirascope.core import openai, prompt_template
```

Comment:
nit: here and elsewhere the imports should be alphabetical, this ordering is a result of auto formatting when working in the mirascope repo due to mirascope being treated as a local import (when it will be treated normally when installed and used normally)